spec/models/like_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
```

Comment:
Let's rewrite some of our descriptions to make our spec readable and easily understood. See my example in `user_spec` that makes proper use of `context` to organize specs better.